src/NzbDrone.Core/Notifications/Ntfy/NtfyProxy.cs

```diff
@@ -78,8 +78,20 @@ public ValidationFailure Test(NtfySettings settings)
             }
             catch (HttpException ex)
             {
-                if (ex.Response.StatusCode == HttpStatusCode.Unauthorized || ex.Response.StatusCode == HttpStatusCode.Forbidden)
+                if (ex.Response.StatusCode is HttpStatusCode.Unauthorized or HttpStatusCode.Forbidden)
                 {
+                    if (!settings.AccessToken.IsNullOrWhiteSpace())
+                    {
+                        _logger.Error(ex, "Invalid token");
+                        return new ValidationFailure("AccessToken", "Invalid token");
+                    }
+
+                    if (!settings.UserName.IsNullOrWhiteSpace() && !settings.Password.IsNullOrWhiteSpace())
+                    {
+                        _logger.Error(ex, "Invalid username or password");
+                        return new ValidationFailure("UserName", "Invalid username or password");
+                    }
+
                     _logger.Error(ex, "Authorization is required");
                     return new ValidationFailure("UserName", "Authorization is required");
```

Comment:
If `AccessToken` is the first field then I think it'd make sense to change this to `AccessToken`